docs/two-way.md

```diff
@@ -82,19 +82,21 @@ The following two-way binds the `<edit-plate>` element’s `plateName` to the `e
 value in the scope.  This allows `plateName` to update if `editing.licensePlate` changes and
 `editing.licensePlate` to update if `plateName` changes.
 
+Click on one of the list items below and watch as its text appears in the input box. You can then edit the text and it will update in the list.
+
 @demo demos/can-stache-bindings/two-way.html
+@codepen
 
 This demo can be expressed a bit easier with the references scope:
 
 @demo demos/can-stache-bindings/reference.html
+@codepen
 
 ## Initialization
 
 When a binding is being initialized, the behavior of what the viewModel and scope properties
-are set to depends on their initial values.
-
-If the viewModel value is `not undefined` and the scope is `undefined`, scope will be set to the viewModel value.
-
-If the viewModel value is `undefined` and the scope is `not undefined`, viewModel will be set to the scope value.
+are set to depends on their initial values:
 
-If both the viewModel and scope are `not undefined`, viewModel will be set to the scope value.
+- If the viewModel value is `not undefined` and the scope is `undefined`, scope will be set to the viewModel value.
```

Comment:
should we use _defined_ instead of _not undefined_?

Reply:
Yeah that sounds better, not undefined is just confusing. I'll change it and submit another PR